src/main/java/edu/ucsb/cs48/s20/demo/entities/ProjectIdea.java

```diff
@@ -16,16 +24,25 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
 
-    @OneToOne
+    @OneToOne(mappedBy = "projectIdea", cascade = CascadeType.REFRESH)
     @JoinColumn(name = "student_id")
     private Student student;
 
     @NotBlank
     private String title;
 
     @NotBlank
+    @Column(columnDefinition = "text")
```

Comment:
Unfortunately this doesn't work with Heroku's postgres; heroku just doesn't seem to think it's anything BUT a `varchar(255)`